### ISSUE TITLE
验证集准确率异常高

### ISSUE THREAD

**Issue by potatojing** [ljynlp/W2NER#124]
用作者提供的config文件跑genia数据集，会发现前几个epoch验证集的准确率比训练集高很多，这是什么原因？

> 2025-04-12 03:27:24 - INFO: Epoch: 0
> /usr/local/lib/python3.11/dist-packages/torch/utils/data/dataloader.py:624: UserWarning: This DataLoader will create 4 worker processes in total. Our suggested max number of worker in current system is 2, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
>   warnings.warn(
> 2025-04-12 03:32:33 - INFO: 
> +---------+--------+--------+-----------+--------+
> | Train 0 |  Loss  |   F1   | Precision | Recall |
> +---------+--------+--------+-----------+--------+
> |  Label  | 0.0718 | 0.3714 |   0.3977  | 0.4219 |
> +---------+--------+--------+-----------+--------+
> /usr/local/lib/python3.11/dist-packages/torch/utils/data/dataloader.py:624: UserWarning: This DataLoader will create 4 worker processes in total. Our suggested max number of worker in current system is 2, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
>   warnings.warn(
> 2025-04-12 03:32:45 - INFO: EVAL Label F1 [0.99883741 0.88873786 0.80859975 0.75128645 0.65076661 0.76485788
>  0.69709544]
> 2025-04-12 03:32:45 - INFO: 
> +--------+--------+-----------+--------+
> | EVAL 0 |   F1   | Precision | Recall |
> +--------+--------+-----------+--------+
> | Label  | 0.7943 |   0.7842  | 0.8172 |
> | Entity | 0.7768 |   0.7657  | 0.7883 |
> +--------+--------+-----------+--------+
> /usr/local/lib/python3.11/dist-packages/torch/utils/data/dataloader.py:624: UserWarning: This DataLoader will create 4 worker processes in total. Our suggested max number of worker in current system is 2, which is smaller than what this DataLoader is going to create. Please be aware that excessive worker creation might get DataLoader running slow or even freeze, lower the worker number to avoid potential slowness/freeze if necessary.
>   warnings.warn(
> 2025-04-12 03:32:58 - INFO: TEST Label F1 [0.99884993 0.89210867 0.7844172  0.71440897 0.70106762 0.74470457
>  0.7611336 ]
> 2025-04-12 03:32:58 - INFO: 
> +--------+--------+-----------+--------+
> | TEST 0 |   F1   | Precision | Recall |
> +--------+--------+-----------+--------+
> | Label  | 0.7995 |   0.7979  | 0.8084 |
> | Entity | 0.7584 |   0.7752  | 0.7423 |
> +--------+--------+-----------+--------+
>
